testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1277,8 +1277,8 @@ public void verifyCompressionCodecsDataProvider()
         assertThat(onTrino().executeQuery("SHOW SESSION LIKE 'iceberg.compression_codec'"))
```

Comment:
What is the purpose of this test ? Not sure why we want to assert on the default here.
Aren't we testing all compression codecs anyway ?

Reply:
Ah, okay got it now.
Could we update the assertion to match only the name and description columns ?

Reply:
I wouldn't modify at least for now. We may want to catch the default session property change here.

Reply:
If the purpose is to ensure that all the supported compression codecs are tested, it doesn't look like we should care about change in the default compression codec here. But not a blocker for this PR.